xbmc/utils/GroupUtils.cpp

```diff
@@ -147,11 +154,14 @@ bool GroupUtils::Group(GroupBy groupBy, const std::string &baseDir, const CFileI
   return true;
 }
 
-bool GroupUtils::GroupAndSort(GroupBy groupBy, const std::string &baseDir, const CFileItemList &items, const SortDescription &sortDescription, CFileItemList &groupedItems, GroupAttribute groupAttributes /* = GroupAttributeNone */)
+bool GroupUtils::GroupAndMix(GroupBy groupBy, const std::string &baseDir, const CFileItemList &items, CFileItemList &groupedItemsMixed, GroupAttribute groupAttributes /* = GroupAttributeNone */)
 {
-  if (!Group(groupBy, baseDir, items, groupedItems, groupAttributes))
+  CFileItemList ungroupedItems;
+  if (!Group(groupBy, baseDir, items, groupedItemsMixed, ungroupedItems, groupAttributes))
     return false;
 
-  groupedItems.Sort(sortDescription);
+  // add all the ungrouped items as well
+  groupedItemsMixed.Append(ungroupedItems);
```

Comment:
Stop even me I am NOT interested stop harassing me
On Sep 14, 2015 2:22 AM, "Sascha Montellese" <notifications@github.com>
wrote:

> In xbmc/utils/GroupUtils.cpp
> <https://github.com/xbmc/xbmc/pull/8009#discussion_r39363810>:
>
> >      return false;
> >
> > -  groupedItems.Sort(sortDescription);
> > +  // add all the ungrouped items as well
> > +  groupedItemsMixed.Append(ungroupedItems);
>
> If you look at the individual commits I first removed the existing
> GroupAndSort() because it wasn't used anywhere. Then I introduced a new
> GroupAndMix() so the overall diff looks a bit odd.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/xbmc/xbmc/pull/8009/files#r39363810>.
>